editor/src/clj/internal/util.clj

```diff
@@ -43,6 +43,18 @@
 (defn conjv [coll x]
   (conj (or coll []) x))
 
+(defn intov
+  ([] [])
+  ([to]
+   {:pre [(or (nil? to) (vector? to))]}
+   (or to []))
+  ([to from]
+   {:pre [(or (nil? to) (vector? to))]}
+   (into (or to []) from))
+  ([to xform from]
+   {:pre [(or (nil? to) (vector? to))]}
+   (into (or to []) xform from)))
+
```

Comment:
`into` defaults to chucking stuff into a linked list if you do `(into nil ...)`. This makes it a bit annoying to use with `update` if you care about the things you put in coming out in the same order.